src/tests/Microsoft.PowerFx.Interpreter.Tests/Helpers/BasicGovernor.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Xunit;
+
+namespace Microsoft.PowerFx.Tests
+{
+    // Custom exception thrown by Governor object. 
+    // Host implements the Governor and can control what's thrown. 
+    // Host also calls eval and controls what's caught. 
+    internal class GovernorException : Exception
+    {
+        public GovernorException(string msg)
+            : base(msg)
+        {
+        }
+    }
+
+    internal class BasicGovernor : Governor
```

Comment:
This is a suggested default implementaiton.
Note that GC.GetAllocatedBytesForCurrentThread() requires NetStandard2.1, but Interpreter uses Net2.0.  Hence this helper class needs to live in the host.